### PR TITLE
Improved docs for std.range.retro

### PR DESCRIPTION
**Pull request by JackStouffer**

### CODE REVIEW

File: std/range/package.d [hunk 1]
```diff
@@ -197,6 +197,14 @@ import std.traits; // CommonType, isCallable, isFloatingPoint, isIntegral,
 Iterates a bidirectional range backwards. The original range can be
 accessed by using the $(D source) property. Applying retro twice to
 the same range yields the original range.
+
+Params:
+    r = a bidirectional range
```
Comment (burner) [inline std/range/package.d]: hm not really happy with the description. maybe something like

r = the bidirectional range to iterate backwards

**Comment by JackStouffer** [comment]
Fixed

File: std/range/package.d [hunk 2]
```diff
@@ -197,6 +197,14 @@ import std.traits; // CommonType, isCallable, isFloatingPoint, isIntegral,
 Iterates a bidirectional range backwards. The original range can be
 accessed by using the $(D source) property. Applying retro twice to
 the same range yields the original range.
+
+Params:
+    r = the bidirectional range to iterate backwards
+
+Returns:
+    A bidirectional range with length if `r` also provides a length. Or,
+    if `r` is a random acess range, then the return value will be random
```
Comment (burner) [inline std/range/package.d]: acess -> access

**Comment by JackStouffer** [comment]
@burner Fixed

**Comment by burner** [comment]
LGTM

**Comment by burner** [comment]
Auto-merge toggled on

**Pull request merged**